etc/less.profile

```diff
@@ -16,6 +16,7 @@ novideo
 shell none
 tracelog
 
+private-bin less
```

Comment:
There is currently a thread (#1450), where someone was using external filters for less for syntax highlighting.
I would guess that also for viewing compressed files external tools (like gzip etc.) are necessary.
Perhaps add a comment that mentions this?